### PR TITLE
Add support unixsocket

### PR DESCRIPTION
**Pull request by ruslan-gennadievich**

### CODE REVIEW

File: test/tcpUnixsocket.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      socket.unref()
+    }
+  })
+})
+
+// This ridiculousness is because when the tests are run via
+// gulp.mocha there's something that causes it to run idefinitely.
+// It doesn't matter that we have close all of the sockets and killed all
+// of the children.
+after(function () { setImmediate(process.exit) })
```
Comment (mcollina) [inline test/tcpUnixsocket.js:65]: I would recommend that every test closes all its open sockets.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "pino-socket",
-  "version": "2.0.0",
-  "description": "A pino 'transport' for writing to a tcp or udp socket",
+  "version": "2.0.1",
```
Comment (mcollina) [inline package.json:3]: please revert this

File: test/tcpUnixsocket.js (hunk 1 above)
Comment (ruslan-gennadievich) [reply test/tcpUnixsocket.js:65]: Yes, i close socket on finally block
`test('unix socket test', (done) => {
  unixSocketTest(done, [], (msg, socket) => {
    try {
      expect(msg).to.contain('"foo":"bar"')
      expect(msg.substr(-1)).to.equal('\n')
      done()
    } catch (e) {
      done(e)
    } finally {
      socket.close()
      socket.unref()
    }
  })
})`

I can delete this?
`after(function () { setImmediate(process.exit) })`

Comment (mcollina) [reply test/tcpUnixsocket.js:65]: yes!

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "pino-socket",
   "version": "2.0.0",
-  "description": "A pino 'transport' for writing to a tcp or udp socket",
+  "description": "A pino 'transport' for writing to a tcp or udp socket or unix socket",
```
Comment (jsumners) [inline package.json:4]:
```suggestion
  "description": "A pino 'transport' for writing to a tcp, udp, or unix socket",
```

**Pull request merged**